internal/services/agent/agent.go

```diff
@@ -0,0 +1,36 @@
+package agent
+
+import (
+	"net/http"
+)
+
+//go:generate mockgen -source=agent.go -destination=mocks/agent.go -package=mocks
```

Comment:
SUGGESTION: it should be in `internal/mockgen.go`

Reply:
++